archive/002.md

```diff
@@ -0,0 +1,58 @@
+Hello EKS News readers! This week's newsletter contains a blog about Kubernetes service discovery using AWS Cloud Map MCS controller, Karpenter on TGIK, tracing traffic through Kubernetes, and more.
+
+## New and notable blogs
+
+[Kubernetes Multi-cluster Service Discovery using the AWS Cloud Map MCS Controller](https://blog.bytequalia.com/kubernetes-multi-cluster-service-discovery-using-the-aws-cloud-map-mcs-controller/)
+
+* MCS services allow you to share and deploy services across multiple clusters.
+* Similar to other network centric approaches such as Cilium’s cluster mesh or Linkerd’s mirrored services
+* AWS Cloud Map MCS is an implementation of the [MCS API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#kep-1645-multi-cluster-services-api), allowing you discover and access services outside a cluster
+* 2 primary scenarios: **Different services each deployed to separate clusters** and **Single service deployed to multiple clusters**
+* The MCS-Controller is responsible for “syncing” services across multiple clusters with Cloud Map serving as the centralized service registry
+* The MCS-Controller release version is [v0.2.2](https://github.com/aws/aws-cloud-map-mcs-controller-for-k8s/releases/tag/v0.2.2) with GA expected in H2 CY2022
+* Includes a tutorial that walks through how setup a multi-cluster service with MCS
+
+## Containers from the Couch
+
+[Using Kubecost to monitor your Kubernetes and infrastructure spend](https://youtu.be/2IJntyJy6vY)  
+We look at how [Kubecost](https://www.kubecost.com/) can be used to track your infrastructure spending and save you money!
```

Comment:
You should also link to the kubectl plugin https://github.com/kubecost/kubectl-cost

Reply:
Done.